doc/development/plugins.rst

```diff
@@ -466,11 +466,13 @@ where
 * :attr:`~.Operation.grad_method`: the gradient computation method; ``'A'`` for the analytic
   method, ``'F'`` for finite differences, and ``None`` if the operation may not be differentiated
 
-* :attr:`~.Operation.grad_recipe`: The gradient recipe for the analytic ``'A'`` method.
-  This is a list with one tuple per operation parameter. For parameter :math:`k`, the tuple is of
-  the form :math:`(c_k, s_k)`, resulting in a gradient recipe of
+* :attr:`~.Operation.grad_recipe`: The gradient recipe for the analytic ``'A'``
+  method. This is a tuple with one nested list per operation parameter. For
+  parameter :math:`k`, the nested list contains elements of the form
+  :math:`[c_i, a_i, s_i]` where :math:`i \in I_{k}` is the index of the term,
```

Comment:
`I` is an index set and `I_{k}` is the set of indices for parameter `k` that index coefficients, scaling factors and shifts. The reason why this would be needed is that for some parameters (e.g., single-qubit rotations), we'd have only two terms, but for others (e.g., controlled rotations), we'd have more than that (four).

Will just leave `where :math:`i` is the index of the term` here.